package.json

```
@@ -83,8 +83,9 @@
   ],
   "scripts": {
     "gendocs": "typedoc --options typedoc.js src/ && node scripts/post-gendocs.mjs",
-    "lint:eslint": "eslint 'src'",
-    "lint:prettier": "prettier -c .",
+    "lint:eslint": "eslint src",
+    "lint:prettier": "prettier --check .",
+    "format": "prettier --write --list-different .",
```

because of https://github.com/enisdenjo/graphql-sse/pull/61#discussion_r1206795511

```suggestion
    "format": "yarn lint:prettier --write",
```

---

Thanks! 😍